src/views/ModelPlan/Collaborators/AddCollaborator/index.tsx

```diff
@@ -125,7 +125,7 @@ const Collaborators = () => {
           input: {
             modelPlanID: modelID,
             userName: username!,
-            teamRole: teamRole!
+            teamRoles: teamRoles!
```

Comment:
Previously we had this as non-nullable because the form element (select) didnt make it possible to choose a null value.  Now with multiselect that is indeed possible.  We dont want to have this non null assertion here unless we have fullproof validation or adjust the form type to more accurately describe this field